UiPath.PowerShell/Util/MaintenanceSettingConverter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using UiPath.Web.Client201910.Models;
+
+namespace UiPath.PowerShell.Util
+{
+    /// <summary>
+    /// Converts maintenance logs into a dynamic objects
+    /// </summary>
+    public static class MaintenanceSettingConverter
+    {
+        private static dynamic resultTyped = new
+        {
+            State = "",
+            MaintenanceLogs = new[] { new { State = "", TimeStamp = new Nullable<DateTime>(DateTime.Now) } },
+            JobsStopped = 0,
+            JobsKilled = 0,
+            SchedulesFired = 0,
+            SystemSchedulesFired = 0
```

Comment:
These need to be updated to match the new schema:

JobStopsAttempted
JobKillsAttempted
TriggersSkipped
SystemTriggersSkipped